website/mod.ts

```diff
@@ -25,6 +26,12 @@ export interface Props {
    * @description These sections will be included on all website/pages/Page.ts
    */
   global?: Section[];
+
+  /**
+   * @title Error Page
+   * @description This page will be used when something goes wrong rendering a page
```

Comment:
This page will be used when something goes wrong beyond section error-boundaries when rendering a page